### PR TITLE
frontend: Design adaptions

### LINKED ISSUE #859: Frontend: small design adaptions
There are a few places where we differ from the design. Most prominent is the transfer history. Olympe gave also some points to improve there. For example the filled dots should not have a border and the text should be ordered slightly differently. Also the text is moving a bit when steps are being completed due to the dots' animation, but it should stay in place.
Where we use teal-light as the color, the design uses actually sea green (#05B0AB).

- [x] align with Olympe 
- [x] implement changes

### PR DESCRIPTION
**Pull request by manuelwedler**
Closes #859 

There were a few places where our implementation of the transfer history differed from the design. Besides, this fixes the dots animation to not move the text around in the history.

Also the tailwind color theme has been reworked to make it align with the figma design.

### CODE REVIEW

File: frontend/src/components/layout/Tabs.vue [hunk 1]
```diff
@@ -52,9 +52,9 @@ const headers = computed(() =>
     click: () => (activeTab.value = tab),
     classes: {
       'cursor-pointer': tab.label != activeTab.value.label,
-      'bg-teal-very-dark/40': tab.label != activeTab.value.label,
-      'text-teal-light': tab.label == activeTab.value.label,
-      'text-teal-light/40': tab.label != activeTab.value.label,
+      'bg-black/40': tab.label != activeTab.value.label,
+      'text-sea-green': tab.label == activeTab.value.label,
+      'text-sea-green/40': tab.label != activeTab.value.label,
     },
```
Comment (GabrielBuragev) [inline frontend/src/components/layout/Tabs.vue:58]: Maybe we can simplify this by creating `inactiveTabClasses` & `activeTabClasses` and then inside the dom element assign the classes based on the condition `header.label == activeTab.label`.
The current approach makes all the header objects carry all this info with them which is unnecessary. 
Also these 4 equality checks are being evaluated each time per header element.

**Review by GabrielBuragev — CHANGES_REQUESTED** [review]
One minor adjustment that i think would be beneficial. Otherwise everything looks good.
Nice UI improvements btw!

File: frontend/src/components/layout/Tabs.vue (hunk 1 above)
Comment (manuelwedler) [reply frontend/src/components/layout/Tabs.vue:58]: Not really related to this PR, but yes, makes it much cleaner. I'll go for it.

**Review by GabrielBuragev — APPROVED** [review]
Perfect!

File: frontend/src/components/layout/Tabs.vue (hunk 1 above)
Comment (GabrielBuragev) [reply frontend/src/components/layout/Tabs.vue]: Shouldn't have requested changes, a simple comment would've been enough, but thanks for understanding the need for this!

**Pull request merged**